apps/src/templates/sectionProgress/summary/VirtualizedSummaryView.jsx

```diff
@@ -30,17 +27,26 @@ class VirtualizedSummaryView extends Component {
     section: sectionDataPropType.isRequired,
     scriptData: scriptDataPropType.isRequired,
     lessonOfInterest: PropTypes.number.isRequired,
-    getLevels: PropTypes.func,
+    levelsByLesson: PropTypes.object,
     onScroll: PropTypes.func,
     jumpToLessonDetails: PropTypes.func.isRequired
   };
 
-  state = {
-    fixedColumnCount: 1,
-    fixedRowCount: 1,
-    scrollToColumn: 0,
-    scrollToRow: 0
-  };
+  constructor(props) {
+    super(props);
+    this.state = {
+      fixedColumnCount: 1,
+      fixedRowCount: 1,
+      scrollToColumn: 0,
+      scrollToRow: 0
+    };
+    this.summaryView = null;
+    this.setSummaryViewRef = this.setSummaryViewRef.bind(this);
+  }
+
+  setSummaryViewRef(ref) {
```

Comment:
NIT: this is such a simple one-liner that stylistically i prefer just using an arrow function in `render`. every single example of the callback method of setting refs that i've seen used an arrow function, and it avoids having to bind this in the constructor, so in total this method adds four more lines than the other.

Reply:
I had this as a one-liner originally, but changed it to match the style guide: https://github.com/airbnb/javascript/tree/master/react#methods

Reply:
i'd argue that the first example in that section is more relevant: "use arrow functions to close over local variables"
```
function ItemList(props) {
  return (
    <ul>
      {props.items.map((item, index) => (
        <Item
          key={item.key}
          onClick={(event) => { doSomethingWith(event, item.name, index); }}
        />
      ))}
    </ul>
  );
}
```

but i agree with your comment below about deciding these things as an engineering team. i will try to find some time soon to draft a proposal for additions to the style guide. in the meantime we can just roll with personal preference! :)